graphql/uniswap/queries/swapsByPoolByAddress.graphql

```diff
@@ -0,0 +1,15 @@
+query swapsByPoolByAddress($pool: String, $userAddress: Bytes) {
```

Comment:
unfortunately looks like we need a second query in order to filter on address

Reply:
just commenting for future that "recipient" will yield better results than "sender" but it is moot in case where we take a fee as the papr controller is both sender and recipient